### DISCUSSION TITLE
Delete button is not appear in Fields – Picklists

### DISCUSSION THREAD

**Discussion by delatdecatsini** [YetiForceCompany/YetiForceCRM#16601] [General]
Hi,
After install on self hosting, with login by adminitrator account, i go to Fields – Picklists, module contact and picklist salutation, the delete button for Salutation Values is not appear when move on the mouse cursor..., only edit icon is appear
I also check the the permissions for delete, edit... its ok

So how to fix that error?
Is it caused by Haven't registered a license yet?
Thank you.

**Comment by rskrzypczak** [comment]
> Is it caused by Haven't registered a license yet?

No, and I have no idea what could be causing it.
Does the cron work for you?

**Reply by delatdecatsini** [reply]
How to check cron work for yetiforce?, i check with wordpress cron is work fine.

**Comment by VictorTWB** [comment]
![Screenshot 2024-01-16 103425](https://github.com/YetiForceCompany/YetiForceCRM/assets/140700741/9773c9cd-1a84-4896-98b2-d60ffacd8475)

Hi, do you mean this? I have the same issue too. Anyone know the possible cause? Thank you.

**Comment by rskrzypczak** [comment]
The inability to delete a dictionary value in the view presented by @VictorTWB  means that the values are hidden somewhere in the code and used in some processes. This is not an error.
